AWSSDKSwiftCLI/Sources/AWSCLIUtils/Process+Utils.swift

```diff
@@ -29,14 +29,14 @@ extension Process {
     
     /// Returns the executable and arguments combined as a string
     var commandString: String {
-        let items = [executableURL?.path] + (arguments ?? [])
+        let items = [urlPath(executableURL)] + (arguments ?? [])
```

Comment:
As you will see below, Foundation `URL` path is different between Mac & Linux, hence we use this wrapper method to get it.